docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
```

Comment:
It's not for restricting. It's for assigning required permissions.

Reply:
Changed heading to `Assign Required Cloud Permissions For Deploying Scalar DL` in c601da6